### ISSUE TITLE
tendrl UI not showing volume details after sometime

### ISSUE THREAD

**Issue by GowthamShanmugam** [Tendrl/gluster-integration#678]
volume utilization function has sys.exit statement, when any problem in fetching volume utilization it stopping to sync thread execution so after sometime volumes are deleted by TTL

**Comment by shtripat** [comment]
This was there because earlier the same was executed as a separate command line tool.
Makes sense to throw and exception now and tackle the same in calling code.

@GowthamShanmugam do you agree?

**Comment by GowthamShanmugam** [comment]
@shtripat yes shubhendu that is correct, I will modify it as per your comment
